Src/DependencyCollector/NetCore/DependencyCollector.csproj

```diff
@@ -0,0 +1,73 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\Common\Common.projitems" Label="Shared" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Signing.props'))\Signing.props" />
+  <Import Project="..\Shared\DependencyCollector.Shared.projitems" Label="Shared" />
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.AI.DependencyCollector</AssemblyName>
+    <VersionPrefix>2.6.1</VersionPrefix>
+    <Authors>Microsoft</Authors>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <TargetFrameworks>net45;netstandard1.6</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
+    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <!--Package Settings-->
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
```

Comment:
so you need to remove corresponding `NuGet.csproj` (https://github.com/Microsoft/ApplicationInsights-dotnet-server/tree/develop/Src/DependencyCollector/NuGet).